01/app.js

```diff
@@ -0,0 +1,24 @@
+const formEl = document.querySelector('form');
+const ulEl = document.querySelector('ul');
+
+formEl.addEventListener('submit', createUser);
+
+function createUser(e) {
+
+    e.preventDefault();
+
+    const firstName = e.target.elements.firstName.value;
+    const lastName = e.target.elements.lastName.value;
+
+    if(firstName.length === 0) {
+        alert('field required');
+    } else if (lastName.length === 0) {
+        alert('field required');
```

Comment:
Wygodniej dla użytkownika zrobić to niezależnie tj. od razu wyświetlić komunikat w obu przypadkach.
Obecnie jak oba są puste - to mamy 1 komunikat. Potem wypełnimy jeden to znów mamy ten sam komunikat - możemy być skołowaniu czy to błąd programu czy nasz.

Lepiej
```
const errors = []
if(firstName.length === 0) {
    errors.push('field required');
} 
if (lastName.length === 0) {
    errors.push('field required');
}

if(errors.lenght) {
     const userName = firstName + ' ' + lastName;
     const liEl = document.createElement('li');
     liEl.classList.add('user-list__person');
     liEl.innerText = userName;
     ulEl.appendChild(liEl);
} else {
    // ...
}
```